### PR TITLE
d_i_n_t_o

### LINKED ISSUE #11: DINTO Error and Warning highlights

### LINKED ISSUE #12: DINTO Logfile Generation

### LINKED ISSUE #13: On-Screen DINTO Reporting

### PR DESCRIPTION
**Pull request by c-brenn**
closes #11.
closes #13.
closes #12.

### CODE REVIEW

File: doc/FEATURES.md [hunk 1]
```diff
@@ -184,12 +184,30 @@ result in no DDIs being reported to the user.
...
-## DINTO Logfile Generation
+### Description
+The system must show errors and warnings on screen when loading a new ontology.
+
+The instructions for loading a new ontology can be found in the [updating dinto]
+document. As mentioned in the [updating dinto] document, loading a new ontology
+requires deploying new versions of system services to production.
+
+If the ontology is malformed, cannot be found or causes any other error, then
+the `docker build` step will fail an appropriate print an error message.
```
Comment (houli) [inline doc/FEATURES.md]: "will fail and print an appropriate error message"

File: doc/FEATURES.md [hunk 2]
```diff
@@ -184,12 +184,30 @@ result in no DDIs being reported to the user.
...
+to production.
+
+## DINTO Logfile Generation - Complete
+
+See [On Screen DINTO Reporting](#on-screen-dinto-reporting---complete)
 
-## DINTO Error and Warning highlights
+## DINTO Error and Warning highlights - Complete
 
+See [On Screen DINTO Reporting](#on-screen-dinto-reporting---complete)
```
Comment (houli) [inline doc/FEATURES.md:212]: It doesn't have the --- complete right now

**Review by houli — CHANGES_REQUESTED** [review]

File: doc/FEATURES.md (hunk 2 above)
Comment (c-brenn) [reply doc/FEATURES.md:212]: ty, fixed

File: doc/FEATURES.md [hunk 3]
```diff
@@ -184,12 +184,31 @@ result in no DDIs being reported to the user.
...
-## DINTO Logfile Generation
+### Description
+The system must show errors and warnings on screen when loading a new ontology.
+
+The instructions for loading a new ontology can be found in the [updating dinto]
+document. As mentioned in the [updating dinto] document, loading a new ontology
+requires deploying new versions of system services to production.
+
+If the ontology is malformed, cannot be found or causes any other error, then
+the `docker build` step will fail and print an appropriate print an error
```
Comment (houli) [inline doc/FEATURES.md]: Still a bit grammar borked :)

File: doc/FEATURES.md [hunk 4]
```diff
@@ -184,12 +184,33 @@ result in no DDIs being reported to the user.
...
 
-## DINTO Logfile Generation
+### Description
+The system must show errors and warnings on screen when loading a new ontology.
+
+The instructions for loading a new ontology can be found in the [updating dinto]
+document. As mentioned in the [updating dinto] document, loading a new ontology
+requires deploying new versions of system services to production.
+
+If their is any problem with the ontology file then the `docker build` step will fail
```
Comment (houli) [inline doc/FEATURES.md]: Ye but "there" though

Comment (c-brenn) [reply doc/FEATURES.md]: Considering `they're`. Any objections?

**Review by houli — APPROVED** [review]

**Pull request merged**